src/js/controller/controller.js

```diff
@@ -51,6 +51,7 @@ Object.assign(Controller.prototype, {
         let _interruptPlay;
         let checkAutoStartCancelable = cancelable(_checkAutoStart);
         let updatePlaylistCancelable = cancelable(function() {});
+        let hasPrimedBeforePlay = false;
```

Comment:
What do you think about flipping this so the flag is read as a command (and we don't have to negate it in the check)?
`let primeBeforePlay = true;`